### PR TITLE
Splitted the chart into three different charts

### PR DESCRIPTION
**Pull request by SeverinAlexB**
I've splittet the main chart into three different charts.
- Loss chart with loss and val_loss
- Accurancy chart with acc and val_acc
- Other chart with size and batch

Now its possible to see the accurancy increase and the loss decrease and not just flat lines.

Are there any intents to push this project further?

Greets
Sebu

### CODE REVIEW

**Comment by zormit** [comment]
Hi Sebubu,

I'm just another user who tried hualos. But thanks for your PR, I approve it (and use it)! :+1:

I would also like to see hualos be developed further. Or are there alternatives?

**Comment by SeverinAlexB** [comment]
You can use the [Tensorboard](https://www.tensorflow.org/versions/r0.7/how_tos/summaries_and_tensorboard/index.html) by Google

**Comment by schlerp** [comment]
i second this, tested and works great.
